UnitedIncome/components
# LockCircleIcon

### Description
> Please provide a brief description of your changes below.

Adds the `LockCircleIcon` component.

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

It renders

### Screenshots
> If this pull request includes a visual change please attach any applicable screenshots.

### Comments
> Any additional comments that will aid with the review process.

- [JIRA Ticket](https://unitedincome.atlassian.net/browse/app-*)
